1_Mechanics/02_GMGuide.md

```diff
@@ -1,82 +1,263 @@
+
+### Creating Your World 
+
+## Section 4: Running a Session with Deck of Adventures
+
+### The Target Card System
+
+As a Game Master, you will set up various challenges and difficulties for the
+Players throughout an Adventure. To resolve these challenges, a Player or GM
+will draw a Target Card, and the opposing force makes a Check and draws a
+card. The level of difficulty of a Check is set by the Difficulty Range,
+which is how far away a pip value can be of the card being drawn relative to
+the value of the Target Card. 
+
+Keep in mind the Target Card System is circular, so an Ace is only 1 away from
+both a King and a 2. 
+
+| DR | -6 | -5 | -4 | -3 | -2 | -1 | 0 | +1 | +2 |
+|---------:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+| Card | 8 | 9 | 10 | J | Q | K | A | 2 | 3 |
 
-> You’ve entered the world of Pelagia as a player, and realized you wanted to become a
-Storyteller, one of the most important parts of Pelagia. This is how our voices spread
-through the heavens. Once known as “Beedle Bards,” Storytellers have taken shape all
-throughout the known Cosmiverse.
-> Respected and honored, Storytellers are welcome at every hearth and home, every tavern
-and table, for the joyful company they bring. They tell of adventures, and entertain
-those around them by allowing them to live, for a time, within those adventures.
-Storytellers are known wide and far for their ability, some even call it Magic, to weave
-a story between multiple listeners and spectators and for them to feel they have lived
-those adventures for themselves.
-> This is the magic you might wield, oh reader. Guides may lie ahead, but the true spirit
-must come from deep within you.
-
-> Humbly,
-> Duke Beedle of Pelagia
-
-As the Game Master, your task is to bring together a group of adventurers and guide them
-through a Story of Pelagia. You can bring together close friends in person, or find new
-friends Virtually through ExampleWebsite.com or in the Discord.
-
-If you’ve been a Game Master elsewhere and are hoping to learn about what’s different
-about being a Game Master of Pelagia, you can skip right to Section 3, “What it’s all
-about”. For everyone else, or those Game Masters who want a refresher, we’ll dive into
-what being a Game Master is all about in Section 2: “Getting Started”. If you or your
-players ever have questions or need a helping hand, never be afraid to reach out to the
-team on Discord `LINK NEEDED` or via GitHub issues.
-
-
-## Getting Started
-
-`MISSING`
-
-## What It’s All About
-
-`MISSING`
-
-## Running a Session with Deck of Adventures
 
 ### Successes and Failures
 
-- Critical Successes have a 1.9% chance of success. They should feel amazing. Make an 
-  effort to rewards these exact matches whenever you can.
-- Major Successes have a 7.7% chance of success. This should still feel pretty good to
-  your players. 
-- Successes.
-- Misses.
+- Successes are cards that are drawn with a pip value within the Diffulty
```

Comment:
How much of this is covered already by the full guide? Need it be repeated? I previously had it pretty sparse to only give the crunch